source/common/config/delta_subscription_state.h

```diff
@@ -98,6 +100,8 @@ class DeltaSubscriptionState : public Logger::Loggable<Logger::Id::config> {
   // Feel free to change to unordered if you can figure out how to make it work.
   std::set<std::string> names_added_;
   std::set<std::string> names_removed_;
+  // Tracks resource aliases in on-demand requests since the previous DeltaDiscoveryRequest we sent.
+  std::set<std::string> aliases_added_;
```

Comment:
I would suggest giving `aliases_added_` a different name. For names_added/removed, it's referring to the delta aspect. It seems like this field is "aliases I'll be requesting resolution of in the next message". Having it named like the delta ones will be a little confusing. Maybe like `aliases_awaiting_resolution_` or something like that? (Well, that one isn't great either, because it's not like their resolution has finished at the instant you clear the set...)

Reply:
done.